examples/with-jest-typescript/package.json

```diff
@@ -8,22 +8,22 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "^5.0.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "next": "^6.0.0",
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0"
   },
   "devDependencies": {
-    "@types/jest": "^22.2.2",
-    "@types/next": "^2.4.8",
-    "@types/react": "^16.0.41",
-    "@types/react-dom": "^16.0.4",
+    "@types/jest": "23.0.0",
+    "@types/next": "2.4.11",
+    "@types/react": "16.3.16",
+    "@types/react-dom": "16.0.5",
     "@zeit/next-typescript": "1.0.1",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "jest": "^22.4.3",
-    "react-addons-test-utils": "^15.6.2",
-    "react-test-renderer": "^16.2.0",
-    "ts-jest": "^22.4.2",
```

Comment:
Doesn't babel 7 make this module obsolete? Since babel now can transpile typescript as well?